README.md

```diff
@@ -160,4 +160,4 @@ dev_tools/format.sh --check  # to verify format
 
 ---
 
-**Note:** This is not an officially supported Google product
\ No newline at end of file
+**Note:** This is not an officially supported Google product
```

Comment:
what is happening to this line?

Reply:
It looks like it's just a difference in line endings. No content change.

Based on my local copy of the file, it looks like the original did not have a carriage return at the end of the line.

Reply:
thanks